src/components/Icons/SVGWrapper/SVGWrapper.tsx

```diff
@@ -11,16 +11,7 @@ export type SVGWrapperProps = SVGWrapperVariants & {
 };
 
 export const SVGWrapper = forwardRef<SVGSVGElement, SVGWrapperProps>(
-  (
-    {
-      className,
-      size,
-      viewBox = "0 0 24 24",
-      color = "iconNeutralDefault",
-      children,
-    },
-    ref
-  ) => {
+  ({ className, size, viewBox = "0 0 24 24", color, children }, ref) => {
```

Comment:
**suggestion**: I think it will break the existing code - can you then migrate at least those components in MacawUI to use `iconNeutralDefault`?